packages/compass-import-export/src/components/import-preview.tsx

```diff
@@ -322,12 +309,7 @@ function ImportPreview({
         data={values}
         columns={gapOrFields.map((field) => {
           if (typeof field !== 'string' && 'path' in field) {
-            return FieldHeader(
-              field,
-              analyzed,
-              onFieldCheckedChanged,
-              setFieldType
-            );
+            return FieldHeader(field, onFieldCheckedChanged, setFieldType);
```

Comment:
Just wondering, are we working around some weird leafygreen limitation here? I got really confused by this "component" when going through code

Reply:
I don't actually know what that is about, to be honest. I found it like that, I think. The diff / blame is a bit incriminating, but I think I just moved it. I can just merge it in?

Reply:
Removed it.